docs(nuxt): Add docs for Cloudflare Nitro plugin (#14219)
getsentry/sentry-docs: docs/platforms/javascript/guides/cloudflare/frameworks/nuxt.mdx

---
title: Nuxt on Cloudflare
description: "Learn how to add Cloudflare instrumentation to your Nuxt app."
---

> If you're running your Nuxt app on Cloudflare Pages, you need to configure the SDK a little differently to the default setup. This guide will show you how to set up the Sentry Nuxt SDK with Nitro's `cloudflare-pages` deployment preset.

Reviewer: ```suggestion
If you're running your Nuxt app on Cloudflare Pages, you'll need to configure the Sentry Nuxt SDK slightly differently than the default setup. This guide will show you how to set up the SDK with Nitro's `cloudflare-pages` deployment preset.
```

## Install

First, install the Sentry Nuxt SDK in your application. We recommend using the Sentry wizard to automatically install the SDK:

```bash
npx @sentry/wizard@latest -i nuxt
```

If the setup through the wizard doesn't work for you, you can also [set up the Nuxt SDK manually](/platforms/javascript/guides/nuxt/manual-setup/).

<Alert>

If you installed the SDK before, make sure that `@sentry/nuxt` version `9.34.0` or newer is installed.

</Alert>

Now you can install the Sentry Cloudflare SDK. First, install the SDK with your package manager:

<PlatformContent includePath="getting-started-install" />

## Setup

<PlatformContent includePath="getting-started-config" />

### Adding the Plugin

To enable Sentry for your Nuxt app on Cloudflare, follow these steps:

1. **Remove the previous server config file:**
   Delete `sentry.server.config.ts` if it exists.

2. **Add a Nitro plugin:**

>    Create a new file in `server/plugins` (for example, `server/plugins/sentry-cloudflare-setup.ts`).

Reviewer 2: I'd just call it `sentry-cloudflare-plugin.ts`, seems most sensible to me. Naming it `sentry.cloudflare.config.ts` makes me think it would just need a Sentry.init in there.
